### PR TITLE
Post vacation cleanup

### PR DESCRIPTION
**Pull request by mambocab**
Just noticed a couple things to clean up while I was reviewing the PRs I missed.

### CODE REVIEW

File: commitlog_test.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def _provoke_commitlog_failure(self):
 
         if self.cluster.version() < "2.1":
             with open(os.devnull, 'w') as devnull:
-                self.node1.stress(['--num-keys=1000000'], stdout=devnull, stderr=subprocess.STDOUT)
+                self.node1.stress(['--num-keys=1M'], stdout=devnull, stderr=subprocess.STDOUT)
```
Comment (ptnapoleon) [inline commitlog_test.py]: Can old stress also support this syntax? I thought it could not. I'll check locally

Comment (ptnapoleon) [inline commitlog_test.py]: In fact, it does not. -1

**Comment by ptnapoleon** [comment]
+1 to changes in user_functions, -1 to commitlog

File: commitlog_test.py (hunk 1 above)
Comment (mambocab) [inline commitlog_test.py]: Oh, pfft. My mistake; I was skimming and didn't think very hard about this. /me puts on dunce cap

**Comment by mambocab** [comment]
Rebased the stress commit out. Once travis is happy I'll merge.

**Pull request merged**